config/app.php

```diff
@@ -35,11 +35,11 @@
     |
     | This value determines the "environment" your application is currently
     | running in. This may determine how you prefer to configure various
-    | services your application utilizes. Should be true in production.
+    | services the application utilizes. Set this in your ".env" file.
     |
     */
 
-    'production' => false,
+    'env' => env('APP_ENV', 'production'),
```

Comment:
`env` is not optional component? How the build feature behaves with this?

Reply:
Yeah, that's true. I meant to mark this as a draft PR.

Trying to think of the best way to do this.

Reply:
I am still not convinced that we should have this.

Reply:
Should I change it to default to `production`, with no `env()`

Reply:
That may be a good idea, yes 👍

Reply:
Will change it soon.